### PR TITLE
[IMP] Allow for context on fetchmail folder configurations to set default values

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Sorry, my mouse makes a false click

File: fetchmail_attach_from_folder/model/fetchmail_server.py [hunk 1]
```diff
@@ -218,6 +220,15 @@ def button_confirm_login(self, cr, uid, ids, context=None):
             connection = this.connect()
             connection.select()
             for folder in this.folder_ids.filtered('active'):
+                try:
+                    folder_context = safe_eval(folder.context or {})
+                except Exception, e:
+                    raise exceptions.ValidationError(
+                        _('Invalid context \"%s\": %s') % (folder.context, e))
```
Comment (sebalix) [inline fetchmail_attach_from_folder/model/fetchmail_server.py]: Is the backslashes really required here?

Comment (StefanRijnhart) [inline fetchmail_attach_from_folder/model/fetchmail_server.py]: Thanks! Indeed, they are not. Code updated.

**Comment by sebalix** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**